### PR TITLE
LPD-24739 - Create an endpoint to retrieve the list of user in a usergroup

### PR DESCRIPTION
**Pull request by matyaswollner**
https://liferay.atlassian.net/browse/LPD-24739

### CODE REVIEW

**Comment by matyaswollner** [comment]
ci:test:sf

File: modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java [hunk 1]
```diff
@@ -1383,6 +1385,24 @@ protected UserAccount testGetUserAccountsPage_addUserAccount(
 		return _addUserAccount(testGroup.getGroupId(), userAccount);
 	}
 
+	@Override
+	protected UserAccount testGetUserGroupUsersPage_addUserAccount(
+			Long userGroupId, UserAccount userAccount)
+		throws Exception {
+
+		UserAccount newUser = _addUserAccount(
```
Comment (smottal) [inline modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java:1393]: can we do userAccount = _addUserAccount? (so reassign to the same variable?)

**Review by smottal — CHANGES_REQUESTED** [review]

**Review by smottal — APPROVED** [review]

**Comment by smottal** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:forward

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**